packages/kit/src/views/ReferFriends/hooks/useWalletBoundReferralCode.tsx

```
@@ -133,13 +147,18 @@ function InviteCode({
+            title: intl.formatMessage({
+              id: ETranslations.global_success,
+            }),
+          });
           onSuccess?.();
         }
       } catch (e) {
         preventClose?.();
       }
     },
-    [onSuccess, verificationCode, wallet?.id, getReferralCodeWalletInfo],
+    [onSuccess, form, wallet?.id, getReferralCodeWalletInfo],
```

ESLint

[error] React Hook useCallback has a missing dependency: 'intl'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)